<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frontViewDialog_list_item_Dialog</class>
 <widget class="QWidget" name="frontViewDialog_list_item_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>54</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_p">
     <property name="text">
      <string>序号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>label_byparent</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_p1">
       <property name="text">
        <string>品名</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>label_byparent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_p5">
       <property name="styleSheet">
        <string notr="true">color:purple;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_p0">
     <property name="text">
      <string>品码</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>label_byparent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_p2">
     <property name="text">
      <string>单价</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>label_byparent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_p3">
     <property name="text">
      <string>数量</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>label_byparent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_p4">
     <property name="text">
      <string>小计</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>label_byparent</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
